packages/editor/src/toolbar.ts

```diff
@@ -86,7 +86,7 @@ export const configureDownloadButton = (editor: Editor, downloadButton: HTMLElem
 /**
  * Creates and returns a dialog to display the editor's model data in a text area.
  */
-export const createExportDataDialog = () => {
+export const createDataDialog = (action: string, editor?: Editor) => {
```

Comment:
I'm not exactly sure what is meant by 'model data'. Is is the model schema? If so, rename accordingly. Also, why did it previously contain 'export' in the name? If my assumption is correct and this is displaying the model schema, rename it to `createShowModelSchemaDialog`. Furthermore why is `action` a string? What are the valid values? Could it be changed to a boolean?

Reply:
To export the schema created by the editor, this data dialog is created. To prevent code duplicate, I modified it a bit. Do I need to create another dialog box for uploading model schema?